config/default.yml

```diff
@@ -2,6 +2,9 @@ AllCops:
   Exclude:
     - bin/yarn
     - db/schema.rb
+    - lib/protobufs/traitify
+    - lib/protobufs/traitify/**
+    - lib/protobufs/traitify/**/*
```

Comment:
There's a chance that this line would be enough (other 2 cases wouldn't be necessary), but I'm not entirely sure